addons/website/static/tests/tours/snippet_translation.js

```diff
@@ -38,6 +38,7 @@ wTourUtils.registerWebsitePreviewTour('snippet_translation_changing_lang', {
     {
         content: "Close the dialog",
         trigger: '.modal-footer .btn-primary',
+        timeout: 30000,
```

Comment:
30 seconds to enter translation mode?

Reply:
Could probably lower this quite a bit but it is indeed quite slow. Not sure how these changes could impact the time it takes to enter translation mode but it was failing consistently on my builds.

Reply:
Isn't it taking only 2 seconds?
```
2023-11-13 14:12:59,774 258 INFO 53624493-saas-16-3-all odoo.addons.website.tests.test_ui.TestUiTranslate.browser: Tour snippet_translation_changing_lang on step: 'Enable translation (trigger: .o_translate_website_container a)' 
2023-11-13 14:12:59,778 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:12:59] "GET /website/lang/en?r=%2Fen HTTP/1.1" 303 - 5 0.001 0.003
2023-11-13 14:12:59,782 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:12:59] "POST /web/dataset/call_kw/website/get_current_website HTTP/1.1" 200 - 5 0.002 0.002
2023-11-13 14:12:59,785 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:12:59] "POST /web/dataset/call_kw/website/search_read HTTP/1.1" 200 - 5 0.001 0.004
2023-11-13 14:12:59,791 258 INFO 53624493-saas-16-3-all odoo.addons.website.tests.test_ui.TestUiTranslate.browser: Tour snippet_translation_changing_lang on step: 'Close the dialog (trigger: .modal-footer .btn-primary)' 
2023-11-13 14:12:59,818 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:12:59] "GET /en HTTP/1.1" 200 - 34 0.008 0.028
2023-11-13 14:12:59,820 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:12:59] "GET /web/bundle/web_editor.assets_wysiwyg?lang=fu_GB HTTP/1.1" 200 - 3 0.000 0.019
2023-11-13 14:12:59,825 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:12:59] "GET /website/force/16?path=%2Fen%3Fedit_translations%3D1 HTTP/1.1" 303 - 7 0.002 0.024
2023-11-13 14:12:59,830 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:12:59] "GET /web/assets/1957-661928c/web_editor.assets_wysiwyg.min.css HTTP/1.1" 200 - 6 0.002 0.005
2023-11-13 14:12:59,835 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:12:59] "GET /web/assets/1932-b0a1ac9/web_editor.assets_wysiwyg.min.js HTTP/1.1" 200 - 6 0.001 0.009
2023-11-13 14:13:00,209 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:13:00] "GET /en?edit_translations=1 HTTP/1.1" 200 - 207 0.040 0.340
2023-11-13 14:13:00,211 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:13:00] "GET /web/bundle/website.assets_wysiwyg?lang=fu_GB HTTP/1.1" 200 - 3 0.000 0.331
2023-11-13 14:13:00,255 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:13:00] "GET /web/assets/1959-4e66d07/website.assets_wysiwyg.min.css HTTP/1.1" 200 - 6 0.001 0.006
2023-11-13 14:13:00,260 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:13:00] "GET /web/assets/2295-985a586/website.assets_wysiwyg.min.js HTTP/1.1" 200 - 6 0.001 0.010
2023-11-13 14:13:00,337 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:13:00] "GET /website/translations/d8254db947719e5beb8b95962598178853d89c35?lang=en_US HTTP/1.1" 303 - 5 0.001 0.003
2023-11-13 14:13:00,399 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:13:00] "GET /website/translations/d8254db947719e5beb8b95962598178853d89c35 HTTP/1.1" 303 - 5 0.000 0.002
2023-11-13 14:13:00,427 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:13:00] "GET /web/bundle/web_editor.assets_wysiwyg?lang=fu_GB&website_id=16 HTTP/1.1" 200 - 3 0.000 0.002
2023-11-13 14:13:00,452 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:13:00] "GET /web/bundle/website.assets_wysiwyg?lang=fu_GB&website_id=16 HTTP/1.1" 200 - 3 0.000 0.003
2023-11-13 14:13:00,616 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:13:00] "POST /website/get_switchable_related_views HTTP/1.1" 200 - 223 0.039 0.096
2023-11-13 14:13:00,620 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:13:00] "POST /web/dataset/call_kw/ir.ui.view/render_public_asset HTTP/1.1" 200 - 7 0.001 0.113
2023-11-13 14:13:01,365 258 INFO 53624493-saas-16-3-all werkzeug: 127.0.0.1 - - [13/Nov/2023 14:13:01] "POST /web/dataset/call_kw/ir.ui.view/render_public_asset HTTP/1.1" 200 - 362 0.087 0.765
2023-11-13 14:13:01,617 258 INFO 53624493-saas-16-3-all odoo.addons.website.tests.test_ui.TestUiTranslate.browser: Tour snippet_translation_changing_lang on step: 'Good job! It's time to <b>Save</b> your work. (trigger: div:not(.o_loading_dummy) > #oe_snippets button[data-action="save"]:not([disabled]))' 
2023-11-13 14:13:01,662 258 INFO 53624493-saas-16-3-all odoo.addons.website.tests.test_ui.TestUiTranslate.browser: Tour snippet_translation_changing_lang on step: 'iframe body:not(.editor_enable)' 
```

Reply:
Turns out it was caused by a different issue where BS modals were clicked too soon after being opened when they did not yet react to clicks, so this is no longer needed.